src/pyrad_proc/pyrad/io/mxpol_config.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suppose I can put METADATA and POLARNAMES in the MCH pyart config file.. what about the RADAR_INFO (i.e lat/lon locations for MCH radars) hard coded in mxpol_reader or rather also in the config file?